plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/completion/SQLCompletionAnalyzer.java

```diff
@@ -937,6 +948,39 @@ private void makeDataSourceProposals(@NotNull Map<String, Object> parameters) th
         }
     }
 
+    @Nullable
+    private List<DBSObject> getTableListFromAlias(DBSObjectContainer sc, @Nullable String token)
```

Comment:
There is no sense of having two almost identical methods. 
getTableFromAlias does the same thing, let's remove it and refactor usages to use this method returning list.

Or maybe it's better to refactor out a common part to a separate method.